src/main/java/gov/usgs/wma/waterdata/collections/CollectionParams.java

```diff
@@ -16,6 +16,7 @@
 	public static final String  PARAM_COLLECTION_ID   = "collectionId";
 	public static final String  PARAM_FEATURE_ID      = "featureId";
 	public static final String  PARAM_TIME_SERIES_ID  = "timeSeriesId";
+	public static final String  PARAM_BEST_TIME_SERIES  = "bestTS";
```

Comment:
I'm not sure why this is part of CollectionParams.  Isn't this a new data endpoiont?